src/Elm/Print/Types.hs

```diff
@@ -110,6 +110,7 @@ elmPrimDoc = \case
     ElmResult l r     -> "Result" <+> elmTypeParenDoc l <+> elmTypeParenDoc r
     ElmPair a b       -> lparen <> elmTypeRefDoc a <> comma <+> elmTypeRefDoc b <> rparen
     ElmTriple a b c   -> lparen <> elmTypeRefDoc a <> comma <+> elmTypeRefDoc b <> comma <+> elmTypeRefDoc c <> rparen
+    ElmQuadruplet a b c d -> lparen <> elmTypeRefDoc a <> comma <+> elmTypeRefDoc b <> comma <+> elmTypeRefDoc c <> comma <+> elmTypeRefDoc d <> rparen
```

Comment:
Elm does not support typles beyond triples.
So the elm code generated by this would not compile.
Let's not encourage users in doing something that's elm doesn't want them to do.